packages/instant/src/components/zero_ex_instant_provider.tsx

```diff
@@ -121,6 +122,17 @@ export class ZeroExInstantProvider extends React.Component<ZeroExInstantProvider
         gasPriceEstimator.getGasInfoAsync();
         // tslint:disable-next-line:no-floating-promises
         this._flashErrorIfWrongNetwork();
+
+        // Analytics
+        analytics.addEventProperties({
+            embeddedHost: window.location.host,
+            embeddedUrl: window.location.href,
+            networkId: state.network,
+            providerName: state.providerState.name,
```

Comment:
What does the providerName look like for fallback?

Reply:
```
    getProviderName(provider: Provider): string {
        const providerTypeIfExists = envUtil.getProviderType(provider);
        if (_.isUndefined(providerTypeIfExists)) {
            return provider.constructor.name;
        }
        return PROVIDER_TYPE_TO_NAME[providerTypeIfExists];
    },
```